web/src/services/ProjectService.ts

```diff
@@ -0,0 +1,41 @@
+import axios, { AxiosResponse } from 'axios';
+import { PostProjectSchema, ProjectSchema } from '../schemas/Project';
+
+const API_URL = 'http://127.0.0.1:8080/projects';
+
+
+async function mapResponse (response: AxiosResponse) {
+    return response.data.map((item: any) => ({
+        id: item.id? item.id : "N/A",
+        name: item.name? item.name : "N/A",
+        description: item.description? item.description : "N/A",
+        active: item.active? "Ativo" : "Inativo",
```

Comment:
Aqui, ao mapear a boolean `active` pro front, podemos verificar se há uma `expire_date`
```suggestion
        active: item.expire_date? "Ativo" : "Inativo",
```